src/models/inbound/backend-requests.ts

```diff
@@ -25,7 +25,7 @@
  --------------
  ******/
 
-import { Logger as SDKLogger, RequestOptions, RequestResponse, request } from '@mojaloop/sdk-standard-components'
+import { Logger as SDKLogger, RequestOptions, RequestResponse, request, requests } from '@mojaloop/sdk-standard-components'
```

Comment:
I know this not an issue in this repo, but I don't like having both `request` and `requests` here.